src/shadowsocks_config.ts

```diff
@@ -338,3 +339,63 @@ export const SIP002_URI = {
     return `ss://${userInfo}@${uriHost}:${port.data}/${queryString}${hash}`;
   },
 };
+
+export interface OnlineConfig {
```

Comment:
This is not an OnlineConfig. This is more like a parsed online config url. Perhaps name it something like `ConfigFetchParams`

Reply:
Done in b3f2f19.